.github/metadata.py

```diff
@@ -827,13 +835,12 @@ def save_metadata(self):
             raise FileNotFoundError(
                 msg,
             )
-        self.output_env_file.write_text(content)
+        self.output_env_file.write_text(content, encoding="utf-8")
 
         if self.debug:
             print(f"--- Content of {self.output_env_file} ---")
-            print(self.output_env_file.read_text())
+            print(self.output_env_file.read_text(encoding="utf-8"))
```

Comment:
Why do you have to force encoding here? Isn't `utf-8` the default?

Reply:
The linter whined that the encoding was missing, and needed. I *only* added comments or code to quiet the linters, and used the simplest comment/code to do so.

Reply:
I had the feeling that encoding was quite tricky that's why my initial intention was to left the default as-is, leaving the implementation details to Python. I never really revisited the topic so thank you for pointing me back to the documentation! :)

So yes, again, you are right, and Pylint is right to suggest forcing the encoding. I was not expecting anyone else using these scripts, and even less on Windows! 😅

Thanks again @rasa for your patience, and let's merge this PR! 🤗

Reply:
Just FYI, even on Unix, utf-8 isn’t the default, if `LC_TYPE` isn’t `C` or `POSIX`, per https://docs.python.org/3/library/os.html#python-utf-8-mode . But in Python 3.15, it will be, per https://peps.python.org/pep-0686/

Reply:
PEP 686 is definitively good news! :)